fuse_optimizers/src/optimizer.cpp

```diff
@@ -482,7 +482,7 @@ void Optimizer::setDiagnostics(diagnostic_updater::DiagnosticStatusWrapper& stat
     return;
   }
 
-  status.summary(diagnostic_msgs::DiagnosticStatus::OK, "Optimizer");
+  status.summary(diagnostic_msgs::DiagnosticStatus::OK, "Optimization converged");
```

Comment:
This messages matches the one we get now in normal conditions. Maybe not too elegant, but we can't overwrite it with `mergeSummary`.

Another option would be to change the `status.message` directly in the children `setDiagnostics()` method.